Components/MapGoogle/assets/marker.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" style="isolation:isolate" viewBox="-50 -50 1300 1300" width="1300" height="1300"><linearGradient id="a" x1="15.451%" y1="14.163%" x2="78.541%" y2="90.773%"><stop offset="0%" stop-color="#ffae00"/><stop offset="100%" stop-color="#d800f4"/></linearGradient><mask id="b"><path d="M0 600C0 268.851 268.851 0 600 0s600 268.851 600 600-268.851 600-600 600S0 931.149 0 600z" fill="#fff"/></mask><path d="M0 600C0 268.851 268.851 0 600 0s600 268.851 600 600-268.851 600-600 600S0 931.149 0 600z" fill="url(#a)" mask="url(#b)" vector-effect="non-scaling-stroke" stroke-width="200" stroke="#FFF" stroke-linecap="square" stroke-miterlimit="3"/></svg>
```

Comment:
Woah, I never saw a marker like this before. I was expecting something like: https://iconmonstr.com/location-1/